test/test_autograd.py

```diff
@@ -8087,6 +8087,8 @@ def test_rnn_backward_to_input_but_not_parameters(self, device):
         out.sum().backward()
         self.assertFalse(s.grad is None or s.grad.abs().sum().item() == 0)
 
+    @unittest.skipIf(IS_ARM64, "ITT is not available on ARM")
+    @unittest.skipIf(not torch.profiler.itt.is_available(), "ITT is required")
     @onlyCPU
```

Comment:
@slgong-fb @malfet Shall I remove this `@onlyCPU` check as well? Only check with `is_available()`?

Reply:
Hi @slgong-fb @malfet , any comments here?

Reply:
I am not sure 100%, but we do not need `onlyCPU`. ITT's scope is covering both CPU and GPU, right? So, as long as ITT is there, we can run this test?

Reply:
Yeah, I agree, as long as ITT is there, we can run this test.